src/Components/ProductBottomBar/ProductBottomBar.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./ProductBottomBar.scss";
+import Arrowdown from "../../Images/arrow-down-white.png";
+
+class ProductBottomBar extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      option: "",
+      click: false,
+    };
+  }
+
+  sizeSelectHandler = (size) => {
+    this.setState({ option: size });
+  };
+
+  arrowClickHandler = () => {
+    const { click } = this.state;
+    this.setState({ click: !click });
+  };
+
+  render() {
+    console.log(this.props.size);
+    return (
+      <div
+        className={
+          this.props.isActive ? "ProductBottomBar" : "ProductBottomBar hide"
+        }
+      >
```

Comment:
`ProductBottomBar` 이라는 className 은 `this.props.isActive` 라는 state 값의 true/false 관계에 상관없이 존재하니 삼항연산자에서 빼줘도 될 것 같습니다~
```suggestion
      <div
        className={`ProductBottomBar ${this.props.isActive ? "" : "hide"}`}
      >
```

Reply:
넵~! 간단하게 수정하였습니다 🤩